### DISCUSSION TITLE
Error message of statusDetails in steps field contains ANSI colors | allure-jest

### DISCUSSION THREAD

**Discussion by yaroslavchaadaev** [allure-framework/allure2#2310] [Questions & Support] [answered]
Hello! @epszaw 

In allure-jest "^2.11.1" test results still contains ANSI colors in statusDetail of steps field.

For example, my test failed with message in statusDetails – seems to be OK:

```
{
    "message": "expect(received).not.toBeInstanceOf(expected)\n\nExpected constructor: not Array\n",
    ...
}
```

While statusDetails field in steps array contains ANSI colors:

```
{
    "message": "\u001b[2mexpect(\u001b[22m\u001b[31mreceived\u001b[39m\u001b[2m).\u001b[22mnot\u001b[2m.\u001b[22mtoBeInstanceOf\u001b[2m(\u001b[22m\u001b[32mexpected\u001b[39m\u001b[2m)\u001b[22m\n\nExpected constructor: not \u001b[32mArray\u001b[39m\n"
}
```

Previous discussion: https://github.com/orgs/allure-framework/discussions/2176

**Comment by yaroslavchaadaev** [comment]
And more addition.

Sometimes my test suites fail with an error:

```
FAIL specs/stage/regression/set_barcodes.spec.js
  ● Test suite failed to run
    TypeError: Cannot read properties of undefined (reading 'replace')
      at removeAnsiColorsFromString (node_modules/allure-jest/src/utils.ts:52:7)
```

For what reason can this happen?

**Comment by epszaw** [comment, marked as answer]
Hello again! [There is a fix](https://github.com/allure-framework/allure-js/pull/848) what should work as you expect.

**Reply by yaroslavchaadaev** [reply]
Cool, thanks!
